providers/amazon/src/airflow/providers/amazon/aws/executors/ecs/ecs_executor.py

```diff
@@ -114,6 +123,32 @@ def __init__(self, *args, **kwargs):
 
         self.run_task_kwargs = self._load_run_kwargs()
 
+    def queue_workload(self, workload: workloads.All, session: Session | None) -> None:
+        from airflow.executors import workloads
+
+        if not isinstance(workload, workloads.ExecuteTask):
+            raise RuntimeError(f"{type(self)} cannot handle workloads of type {type(workload)}")
+        ti = workload.ti
+        self.queued_tasks[ti.key] = workload
+
+    def _process_workloads(self, workloads: list[workloads.All]) -> None:
+        from airflow.executors.workloads import ExecuteTask
+
+        # Airflow V3 version
+        for w in workloads:
+            if not isinstance(w, ExecuteTask):
+                raise RuntimeError(f"{type(self)} cannot handle workloads of type {type(w)}")
+
+            # TODO: AIP-72 handle populating tokens once https://github.com/apache/airflow/issues/45107 is handled.
+            command = [w]
+            key = w.ti.key
```

Comment:
Tokens is done now btw.

Reply:
Fair, must have happened in main while I was branched. I'll drop that

Reply:
Revisiting this, I've removed the comment, but I'm not actually sure what work needs to be done. That TODO was just lifted from @amoghrajesh's work on the kubernetes executor (and I still see the TODO there). I assume the token is just passed through with the workload to the supervisor?

Reply:
People are busy and I haven't heard back on this one. I can see the tokens being passed through so I think no action is needed here other than dropping that comment. Resolving this conversation